package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "serverless-plugin-localstack",
```

Comment:
Mainly for two reasons:

1) It matches the actual repository name.
2) I took a look at several other plugins listed on the serverless plugins repo and there were a couple of patterns I found: the npm install was the same as the repo name, or repo-name-plugin.

Since this plugin hasn't been published to NPM yet it seems like a good time to propose a name change.